cms/static/cms/js/require-config.js

```diff
@@ -1,3 +1,14 @@
+// MathJax Fast Preview was introduced in 2.5. However, it
+// causes undesirable flashing/font size changes when
+// MathJax is used for interactive preview (equation editor).
+// Setting processSectionDelay to 0 (see below) fully eliminates
+// fast preview, but to reduce confusion, we are also setting
+// the option as displayed in the context menu to false.
+// When upgrading to 2.6, check if this variable name changed.
+window.MathJax = {
```

Comment:
I'll check that window exists, like we did in the LMS requireJS config (though oddly enough, this seems to be just fine with optimizer...).